src/shared/constants/cookie-age.constant.ts

```diff
@@ -0,0 +1 @@
+export const REFRESH_TOKEN_COOKIE_AGE = 30 * 24 * 60 * 60 * 1000;
```

Comment:
Make sure to align it with the env variable.

Reply:
installed `ms` lib for direct conversions of dates in string format.